src/invoker.js

```diff
@@ -235,6 +238,47 @@ class OpenWhiskInvoker {
         });
     }
 
+    getFullActionName() {
+        return `/${this.wskProps.namespace}/${this.actionName}`;
```

Comment:
will this work for actions in a package? (I assume these are coming from the action context, so the answer to the q would be affirmative.)

Reply:
`this.actionName` includes the package if there is one.

I never know what to call these various entities, namespace+package+action, only action, package+action... 😉